ods_ci/tests/Resources/Page/OCPDashboard/OperatorHub/InstallODH.robot

```diff
@@ -17,7 +17,7 @@ Open OperatorHub
     OperatorHub Should Be Open
 
 Install ODH Operator
-    Install Operator    opendatahub
+    Install Operator    ${OPERATOR_DEPLOYMENT_NAME}
```

Comment:
shouldnt this be ${OPERATOR_NAME}, as we have in the Uninstall ODH Operator keyword?